syncer/checkpoint.go

```diff
@@ -621,19 +620,24 @@ func (cp *RemoteCheckPoint) Load(tctx *tcontext.Context, schemaTracker *schema.T
 		if err != nil {
 			return terror.WithScope(terror.DBErrorAdapt(err, terror.ErrDBDriverError), terror.ScopeDownstream)
 		}
-		gset, err := gtid.ParserGTID(cp.cfg.Flavor, binlogGTIDSet)
-		if err != nil {
-			return err
+		var gset gtid.Set
+		if len(binlogGTIDSet) == 0 {
+			gset = gtid.MinGTIDSet(cp.cfg.Flavor)
+		} else {
+			gset, err = gtid.ParserGTID(cp.cfg.Flavor, binlogGTIDSet)
```

Comment:
`gtid.ParserGTID` with `len(binlogGTIDSet) == 0` seems fine.

Reply:
yes, removed in 8132f8f